src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV2Spec.scala

```diff
@@ -406,7 +408,7 @@ class ResourceRoutesV2Spec extends FlatSpec with Matchers with TestSupport with
       accessToCurrentParent = false)
 
     Delete(s"/api/resources/v2/${defaultResourceType.name}/${fullyQualifiedChildResource.resourceId.value}/parent") ~> samRoutes.route ~> check {
-      status shouldEqual StatusCodes.NotFound
+      status shouldEqual StatusCodes.Forbidden
     }
   }
 
```

Comment:
I think we're missing a test for erroring when a parent is set on a resource, but the parent resource does not exist

Reply:
can you add what is missing?